src/victorops.coffee

```diff
@@ -187,6 +187,8 @@ class VictorOps extends Adapter
       for item in data.PAYLOAD.TIMELINE_LIST
         try
           if item.ALERT
+            @robot.brain.set item.ALERT["INCIDENT_NAME"], item.ALERT
```

Comment:
ENTITY_STATE_NOTIFY_MESSAGEs transmit incident state transitions, and may be more appropriate for your use.  As in incident is resolved, it may also provide an opportunity to clean up the cache, rather than have it grow endlessly.  We have many customers with many different levels of incident frequency, and such an unbounded cache may eventually become a problem for heavy users.